docs/unit10.mdx

```diff
@@ -23,7 +25,7 @@ Most open source projects have their code on GitHub.
 This makes it extremely easy for people to contribute to them. 
 
 :::note Fun Fact
-ICSSC projects are all open sourced. Over 50 students have contributed to the projects!
+ICSSC projects are all open sourced. Over 100 students have contributed to the projects!
```

Comment:
It's actually 95 but it will be over 100 soon lol

Reply:
Fair enough 😄